lua/lspconfig/server_configurations/julials.lua

```diff
@@ -1,5 +1,54 @@
 local util = require 'lspconfig.util'
 
+local function activate_env(path)
+  local bufnr = vim.api.nvim_get_current_buf()
+  local julials_clients = util.get_lsp_clients { bufnr = bufnr, name = 'julials' }
+  assert(
+    #julials_clients > 0,
+    'method julia/activateenvironment is not supported by any servers active on the current buffer'
+  )
+  local julia_project_files = { 'Project.toml', 'JuliaProject.toml' }
+  local function _activate_env(environment)
+    if environment then
+      for _, julials_client in ipairs(julials_clients) do
+        julials_client.notify('julia/activateenvironment', { envPath = environment })
+      end
+      vim.notify('Julia environment activated: \n`' .. environment .. '`', vim.log.levels.INFO)
+    end
+  end
+  if path then
+    path = vim.fs.normalize(path)
+    local found_env = false
+    for _, project_file in ipairs(julia_project_files) do
+      if util.path.exists(util.path.join(path, project_file)) then
+        found_env = true
+        break
+      end
+    end
+    if not found_env then
+      vim.notify('Path is not a julia environment: \n`' .. path .. '`', vim.log.levels.WARN)
+      return
+    end
+    _activate_env(path)
+  else
+    local depot_paths = vim.env.JULIA_DEPOT_PATH and vim.split(vim.env.JULIA_DEPOT_PATH, ':')
```

Comment:
Is ":" the path separator that Julia uses on Windows?

This is why I'm so reluctant to merge these non-trivial command wrappers. They hide numerous subtle decisions that lead to bug reports later.

Reply:
Yes that is the same path separator in windows as well. I apologize for the code doubling, it was me incorrectly implementing things the first time. But this is for Windows as well